.travis.yml

```diff
@@ -4,9 +4,18 @@ php:
   - 5.3
   - 5.4
   - 5.5
+  - 5.6
+  - 7.0
+  - hhvm
 
-before_script:
-  - curl -s http://getcomposer.org/installer | php
-  - php composer.phar install --dev
+## Cache composer
+sudo: false
 
-script: phpunit
\ No newline at end of file
+cache:
+  directories:
+    - $HOME/.composer/cache
+
+install:
+  - travis_retry composer install --prefer-dist --no-interaction
+
+script: vendor/bin/phpspec run
```

Comment:
Yes, because it doesn't have any phpunit tests, right? At least, I could only run phpspec locally